pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/api/ComponentImpl.java

```diff
@@ -1145,18 +1070,20 @@ public Response triggerFunction(final String tenant,
                 producer.closeAsync();
             }
         }
+        return null;
```

Comment:
we shouldn't be returning null here.  All cases should be covered

Reply:
will fix it

Reply:
fixed